MuxPhp/Models/CreateLiveStreamRequest.php

```diff
@@ -160,6 +165,7 @@ public function __construct(array $data = null)
     {
         $this->container['playback_policy'] = isset($data['playback_policy']) ? $data['playback_policy'] : null;
         $this->container['new_asset_settings'] = isset($data['new_asset_settings']) ? $data['new_asset_settings'] : null;
+        $this->container['reconnect_window'] = isset($data['reconnect_window']) ? $data['reconnect_window'] : 60;
```

Comment:
if the default in our system is 300, why are we defaulting to 60 here?

Reply:
Fixed thanks!